libnmstate/validator.py

```diff
@@ -250,9 +250,20 @@ def _assert_vxlan_has_missing_attribute(state, *attributes):
 
 def _assert_vlan_filtering_trunk_tags(ports_state):
     for port_state in ports_state:
-        trunk_tags = port_state.get(LB.Port.VLAN_SUBTREE, {}).get(
-            LB.Port.Vlan.TRUNK_TAGS, []
-        )
+        port_vlan_state = port_state.get(LB.Port.VLAN_SUBTREE, {})
+        if not port_vlan_state:
+            return
```

Comment:
Looks like a bug to me.

Reply:
I just wanted to bail out early if there's no port vlan configuration.

Is it better now ?

Reply:
You can bail out for a specific port, but not for all because of one.

Reply:
touchè.